tools/build/configure.py

```diff
@@ -69,7 +69,7 @@ def write_ninja_rules(
 
     CPPFLAGS_272 = CPPFLAGS_COMMON + " -nostdinc"
 
-    CPPFLAGS_EGCS = CPPFLAGS_COMMON + " -D__USE_ISOC99 -DBBPLAYER -nostdinc"
+    CPPFLAGS_EGCS = CPPFLAGS_COMMON + " -D__USE_ISOC99 -nostdinc"
```

Comment:
is there a reason we can't just always define BBPLAYER for this compiler like we used to?

Reply:
I've moved that define to use for both the C code and HASM code inside the `os/` folder, and into making it easier to create a bootable iQue version for N64 emulators and into real hardware.